social-network-api/src/main/java/fs/socialnetworkapi/security/JwtFilter.java

```diff
@@ -0,0 +1,63 @@
+package fs.socialnetworkapi.security;
+
+import fs.socialnetworkapi.entity.User;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.http.HttpHeaders;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.web.authentication.WebAuthenticationDetailsSource;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+import java.util.Optional;
+
+@Log4j2
+@Component
+@RequiredArgsConstructor
+public class JwtFilter extends OncePerRequestFilter {
+
+  private static final String BEARER = "Bearer ";
+  private static final String AUTH_HEADER = HttpHeaders.AUTHORIZATION;
+
+  private final SecurityService tokenService;
+
+  private Optional<String> extractTokenFromRequest(HttpServletRequest rq) {
```

Comment:
private methods should be under the public/protected. Or try to place the methods in order that you see them in the code.